chidimo/Quick-Credit
# #165839916 Update UI templates

#### What does this PR do?

Update the site UI templates

#### Description of Task to be completed?

Update the site UI templates
Update the navbar
Add separate pages for creating loan application and editing user profile

#### How should this be manually tested?

Simply clone the repo. Navigate to the root and open index.html with a browser. Then manually navigate to each page to see the outputs.

#### Any background context you want to provide?

Nil

#### What are the relevant pivotal tracker stories?

[165839916](https://www.pivotaltracker.com/story/show/165839916)

#### Screenshots (if appropriate)

Nil

#### Questions:

Nil
